spec/core_functions/modules/math/clamp.hrx

```diff
@@ -0,0 +1,171 @@
+<===>
+================================================================================
+<===> options.yml
```

Comment:
I think this should go in `spec/core_functions/math` even though it uses the modules, so it's in the same place as the other core function tests.